backend/Makefile

```diff
@@ -1,4 +1,4 @@
-SHELL := /bin/bash
+SpirunnerHELL := /bin/bash
```

Comment:
What's this?

Reply:
Looks like an accidental paste typo?

Reply:
Embarrassing. Accidental vim `.` that got in without me noticing while fixing the apirunner target